pkg/slices/contains.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2021 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package slices
+
+func ContainsString(str string, list []string) bool {
```

Comment:
There is a similar function in https://github.com/fybrik/fybrik/blob/ac8ea647b849bb028cbea636e86422507197c734/manager/controllers/utils/utils.go#L50
I agree it belongs in the package but please delete the duplicated one and replace the references accordingly.